src/pg_to_evalscript/evalscript.py

```diff
@@ -119,6 +121,11 @@ def determine_output_dimensions(self):
     def set_output_dimensions(self, output_dimensions):
         self._output_dimensions = output_dimensions
 
+    def set_input_bands(self, input_bands):
```

Comment:
This method should probably be described in the `README.md`

Reply:
Indeed, added.

Reply:
Thanks